packages/nuxt/src/core/nuxt.ts

```diff
@@ -60,6 +66,9 @@ export function createNuxt (options: NuxtOptions): Nuxt {
   return nuxt
 }
 
+// TODO: update to nitro import
+const fallbackCompatibilityDate = '2024-04-03' as DateString
```

Comment:
It is a fixed thing (lasted release of Nitro that didn't had the compat date probably you can add the comment simply)

Reply:
yes, I will update when merging to 3.x branch, but current nitro-nightly doesn't have it